common/src/main/java/com/wynntils/services/itemfilter/type/ItemProviderType.java

```diff
@@ -63,5 +62,40 @@ public enum ItemProviderType {
     /**
      * A filter for items that have a profession.
      */
-    PROFESSION
+    PROFESSION,
+
+    /**
+     * Special filter for territory items.
+     */
+    TERRITORY(true);
+
+    /**
+     * Whether this item provider type is special.
+     * Special providers are not included in {@link ItemProviderType#ALL}.
+     */
+    private final boolean isSpecial;
+
+    ItemProviderType() {
+        this(false);
+    }
+
+    ItemProviderType(boolean isSpecial) {
+        this.isSpecial = isSpecial;
```

Comment:
"special" makes me nervous. Why do you need this? Is it just so the personal storage container and guild bank should not have the territory filter?

Reply:
Nothing, except the territory menus should have them.